# deps-recommend-update-rax

Rax version < 1.0 , recommend to update Rax.

## Rule Details

Examples of **incorrect** code for this rule:

```json
{
  "dependencies": {
    "rax": "^0.6.0"
  }
}
```

Examples of **correct** code for this rule:

```json
{
  "dependencies": {
    "rax": "^1.1.0"
  }
}
```